src/main/resources/config/defaults.json

```diff
@@ -1,561 +1,559 @@
-				"recycle": 1000000
+				"recycle" : 1000000
 			},
 			// Specifies whether to recycle the load tasks or not
-			"recycle": false,
+			"recycle" : false,
 			// Specifies whether to retry the failed I/O tasks or not
-			"retry": false,
+			"retry" : false,
 			// Defines whether to shuffle or not the items got from the item input,
 			// what should make the order of the I/O tasks execution randomized
-			"shuffle": false,
+			"shuffle" : false,
 			// The operation to process the items, may be "noop", "create", "update", "read" or "delete"
-			"type": "create"
+			"type" : "create"
 		},
 		// The global count of the service threads. 0 means automatic value
-		"service": {
-			"threads": 0
+		"service" : {
+			"threads" : 0
 		},
-		"step": {
+		"step" : {
 			// The test step id. Generated automatically if not specified (null).
 			// Specifies also the logs sub directory path: log/<STEP_ID>/
-			"id": null,
+			"id" : null,
 			// The internal flag used to determine if the id was automatically generated or not
-			"idAutoGenerated": false,
-			"limit": {
+			"idAutoGenerated" : false,
+			"limit" : {
 				// The maximum size of the data items to process. 0 means no size limit.
-				"size": 0,
+				"size" : 0,
 				// The maximum time to perform a load step. 0 means no time limit
-				"time": 0
+				"time" : 0
 			},
-			"node": {
+			"node" : {
 				// Distributed mode: the list of the Mongoose node IPs or hostnames,
 				// may include port numbers to override the default port number value.
 				// Standalone mode is used if empty (default behaviour).
-				"addrs": [
+				"addrs" : [
 				],
 				// Distributed mode: the common port number to connect the Mongoose nodes
-				"port": 1099
+				"port" : 1099
 			}
 		}
 	},
-	"output": {
+	"output" : {
 		// Use colored standard output flag
-		"color": true,
-		"metrics": {
-			"average": {
+		"color" : true,
+		"metrics" : {
+			"average" : {
 				// The time period for the load step's metrics console output.
 				// 0 means to not to output the metrics to the console
-				"period": "10s",
+				"period" : "10s",
 				// Persist the average (periodic) metrics if true
-				"persist": true,
-				"table": {
-					"header": {
+				"persist" : true,
+				"table" : {
+					"header" : {
 						// Output the metrics table header every N rows
-						"period": 20
+						"period" : 20
 					}
 				}
 			},
-			"summary": {
+			"summary" : {
 				// Output the results.xml file used as a PerfDb input if true
-				"perfDbResultsFile": false,
+				"perfDbResultsFile" : false,
 				// Persist the summary (total) metrics if true
-				"persist": true
+				"persist" : true
 			},
-			"trace": {
+			"trace" : {
 				// Persist the information about each load operation if true
-				"persist": false
+				"persist" : false
 			},
 			// The concurrency threshold to enable intermediate statistics calculation,
 			// 0 means no threshold
-			"threshold": 0
+			"threshold" : 0,
+			"quantiles" : [
```

Comment:
but where is a median??

Reply:
the median is always calculated and there is a separate method for it.

Reply:
Ok